package.json

```diff
@@ -93,6 +93,8 @@
     "@microsoft/loader-load-themed-styles": "2.0.17",
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30313&commitId=f6060b3c4fa54b008afec6c2b96ad7805f0a7432&env=prod&clientType=FLUENTUI)


  <details><summary><b>Label</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Label Example Content Customization Shorthand.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>